app/controllers/api/v1/admin/invitations_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Admin
+      class InvitationsController < ApiController
+        before_action only: %i[create] do
+          ensure_authorized('ManageUsers')
+        end
+
+        # POST /api/v1/admin/invitations
+        def create
+          params[:invitations][:emails].split(',').each do |email|
+            invitation = Invitation.find_or_initialize_by(email:, provider: current_provider).tap do |i|
+              i.updated_at = Time.zone.now
+              i.save!
+            end
+
+            UserMailer.with(
+              email:,
+              name: current_user.name,
+              signup_url: root_url(inviteToken: invitation.token)
+            ).invitation_email.deliver_later
+          rescue StandardError => e
```

Comment:
This is to ensure the completion of the task even when failing to send an email to some users?

Reply:
Correct